app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def calculate_team_total_rating(players: list) -> float:
+    return sum([player.get_rating() for player in players])
```

Comment:
in the checklist to this task written about redundant square brackets.
https://github.com/inrock1/py-elves-and-dwarves/blob/master/checklist.md